filebeat/inputsource/common/streaming/listener.go

```diff
@@ -60,8 +63,30 @@ var (
 		"delimiter": FramingDelimiter,
 		"rfc6587":   FramingRFC6587,
 	}
+
+	framingTypeNames []string
 )
 
+func init() {
+	framingTypeNames := make([]string, 0, len(framingTypes))
+	for t := range framingTypes {
+		framingTypeNames = append(framingTypeNames, t)
+	}
+}
```

Comment:
`availableFramingTypesErrFormat = fmt.Sprintf("invalid framing type %%q, the supported types are [%v]", strings.Join(framingTypeNames, ", "))`